tests/Tests/ORM/Mapping/AttributeDriverTest.php

```diff
@@ -131,3 +131,4 @@ class AttributeTransientAttribute implements MappingAttribute
 class AttributeTransientClass
 {
 }
+
```

Comment:
Unintended change I suppose? Please fixup the corresponding commit.

Reply:
It's where I first wrote the tests until I decided to test all mapping methods.